README.md

```diff
@@ -1 +1,180 @@
+- ![Render](https://img.shields.io/badge/Render-%46E3B7.svg?style=for-the-badge&logo=render&logoColor=white)
+
+<!-- Features -->
+
+### 💎 Key Features <a name="key-features"></a>
+
+- Generate a comprehensive Folder Structure for React App 💻
+- Understand the concept of setting up a project with redux 📈
+- Beautifully designed and easy-to-use 🎨
+- Continuous Monitoring 📸
+- Development Tools : Auto refresh and Hot Reload 🔗
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+***Live*** :  [Math Magician](https://63f89b28ed3dd406ccf11729--ahr-React-Bookstore.netlify.app/).
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need [Node.js](https://nodejs.org/en/) installed on your machine.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-project
+  git clone git@github.com:roniy68/react-bookstore.git .
+```
+
+### Install
+
+Install the dependencies with:
+
+```sh
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+
+<!-- ### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm run test
+``` -->
+
+### Deployment
+
+You can deploy this project using:
+
+```sh
+  npm run build
+```
+This will create a production-ready build of your website in `build/` folder, which you can use to deploy on a static site server.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 📧 Contact <a name="authors"></a>
+
+I am always looking for ways to improve my project. If you have any suggestions or ideas, I would love to hear from you.
+
+[![Github](https://img.shields.io/badge/GitHub-673AB7?style=for-the-badge&logo=github&logoColor=white)](https://github.com/roniy68)
+[![Twitter](https://img.shields.io/badge/Twitter-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white)](https://twitter.com/ahroniy)
+[![Linkedin](https://img.shields.io/badge/LinkedIn-0077B5?style=for-the-badge&logo=linkedin&logoColor=white)](https://linkedin.com/in/ahroniy)
+[![Gmail](https://img.shields.io/badge/Gmail-D14836?style=for-the-badge&logo=gmail&logoColor=white)](mailto:ahmed.hasan.rony@gmail.com)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [x] **Mobile UI**
+- [ ] **Visualize contribution**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+I welcome any and all contributions to my website! If you have an idea for a new feature or have found a bug, please open an issue or submit a pull request.
+
+Feel free to check the [issues page](../../issues/).
+
```

Comment:
- [ ] Please fix your issues page link because it is broken :thinking: Kindly fix it to make your readme file descriptive and professional :+1: